/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 1] Operation not permitted
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 2] No such file or directory
/Applications/Xcode.app/Contents/Developer/usr/bin/python3: can't open file 'monitor_supabase_providers.py': [Errno 2] No such file or directory
